readme.md

```diff
@@ -116,7 +148,9 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
+Instead of bringing the two branches back together (as with a merge) a rebase rewrites the git commits into one single string line, which allows for a cleaner and more linear project history.
 
+I'm not a fan though (at least not yet), it seems less accurate in tracing what actually occurred, and (at least until I know it better), seems less safe in terms of correctly resolving potential conflicts.
```

Comment:
:+1:

I agree.  I am hesitant to rewrite history.  Except when using `git pull --rebase` on master.  It is relatively safe, avoids a (usually) extraneous merge commit, and I usually feel good about the results.